src/Microsoft.DotNet.ProjectJsonMigration/ProjectMigrator.cs

```diff
@@ -30,9 +33,73 @@ public ProjectMigrator(IMigrationRule ruleSet)
             _ruleSet = ruleSet;
         }
 
-        public void Migrate(MigrationSettings migrationSettings)
+        public void Migrate(MigrationSettings rootSettings, bool skipProjectReferences = false)
+        {
+            if (rootSettings == null)
+            {
+                throw new ArgumentNullException();
+            }
+
+            MigrateHelper(rootSettings);
```

Comment:
can we change the name of this function to reflect it's intention? What does the helper do?